packages/@tailwindcss-cli/src/commands/build/index.ts

```diff
@@ -255,6 +266,44 @@ export async function handle(args: Result<ReturnType<typeof options>>) {
   }
 }
 
+async function createWatchers(globs: GlobEntry[], handle: (files: string[]) => void) {
+  let watchers = disposables()
+  let files = new Set<string>()
+
+  let d = disposables()
+  function flush() {
```

Comment:
Mega nit but `flush` feels like it should do something synchronously. Maybe prefixing with "enqueue" could help. What do you think?